wolfcrypt/src/port/kcapi/kcapi_ecc.c

```diff
@@ -179,6 +179,10 @@ int KcapiEcc_Sign(ecc_key* key, const byte* hash, word32 hashLen, byte* sig,
                   word32* sigLen)
 {
     int ret = 0;
+    unsigned char* buf_aligned = NULL;
+    unsigned char* hash_aligned = NULL;
+    unsigned char* sig_aligned = NULL;
+    size_t pageSz = sysconf(_SC_PAGESIZE);
```

Comment:
Recommend an explicit cast to `size_t` here since it is a long. All builds fine for me. Should we have the customer also test before we merge?